manifests/stage-multus-cni/0050-multus-ds.yml

```diff
@@ -36,6 +36,10 @@ spec:
           args:
             - "--cni-version=0.4.0"
             - "--multus-conf-file={{- if .CrSpec.Config -}}/tmp/multus-conf/00-multus.conf{{- else -}}auto{{- end -}}"
+          lifecycle:
```

Comment:
@Mmduh-483 
can you add a comment here explaining why we do that with a link to the multus issue  ?

Reply:
Added